packages/types/src/payment/mutations.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * TODO
+ */
+
+export interface CreatePaymentCollectionDTO {}
+
+export interface UpdatePaymentCollectionDTO {}
+
+export interface CreatePaymentDTO {
+  amount: number
+  currency_code: string
+  provider_id: string
+  data: Record<string, unknown>
+
+  cart_id?: string
+  order_id?: string
+  order_edit_id?: string
+  customer_id?: string
+}
+
+export interface UpdatePaymentDTO {
+  cart_id?: string
```

Comment:
question: should we be able to update the props that are part of the creation too?

Reply:
Yep, I will revisit this in the followup PR, just wanted to add basic interfaces needed for the service